ResearchKit/Common/ORKImageCaptureStepViewController.m

```diff
@@ -299,11 +299,13 @@ - (void)handleError:(NSError *)error {
 }
 
 - (void)setCapturedImageData:(NSData *)capturedImageData {
+    BOOL shouldRemoveFile = capturedImageData != nil;
```

Comment:
What happens if we skip the step? Is `capturedImageData` also nil? If so, what's is the impact then of not removing the image?

Reply:
Skipping means that there wasn't any image data associated. I've checked and it behaves as previously.